bin/ingest-gisaid

```diff
@@ -157,6 +157,17 @@ main() {
     ./bin/upload-to-s3 ${silent:+--quiet} data/gisaid/sequences.fasta "$S3_DST/sequences.fasta.xz"
 
     ./bin/clean
+
+    if [[ $fetch == 1 && "$branch" == master ]]; then
```

Comment:
Same question here if you maybe need to add quotes to match other logic referencing variables?
```suggestion
    if [[ "$fetch" == 1 && "$branch" == master ]]; then
```